src/bin/CMakeLists.txt

```diff
@@ -47,14 +47,13 @@ add_executable(
 )
 target_link_libraries(
     hyriseConsole
+
+    PRIVATE
     hyriseBenchmarkLib
     ncurses
-    ${READLINE_LIBRARY}
 )
-
-target_include_directories(
+target_link_libraries_system(
     hyriseConsole
 
-    SYSTEM PUBLIC
-    ${READLINE_INCLUDE_DIR}
+    readline
```

Comment:
Does readline need to be a system library?

Reply:
Can be a non-system lib